### ISSUE TITLE
Feature-Request: Store lyrics in mp3/flac-tags

### ISSUE THREAD

**Issue by desputinski** [clementine-player/Clementine#5598]
Hi,
in my point of view it would be great to access song lyrics also without internet connection.
So I think it would be great if there was a batch capability to download all lyrics for songs automatically (like the album cover downloader) and add the lyrics to the tags of the mp3 or flac files.
That would be a real "killer feature" :)

Best regards
desputin

**Comment by aezumezu** [comment]
Yea, that would be great. To add to @desputinski's request, it would be nice to be able to keep the lyrics always on top. Just a mini window with a _"transparent"_ background.

**Comment by TheBlackKoala** [comment]
Yes an option for auto-saving lyrics in song-tags would be perfect!

**Comment by desputinski** [comment]
Yep. The problem seems to be that clementine development is quite inactive if I'm correct. Which is a shame, as it is the best music player out there in my point of view.

**Comment by TheBlackKoala** [comment]
Depending on what system you are on MediaHuman's LyricsFinder might work - i'm on arch and used debtap to get it working :)
